### PR TITLE
applecamerad: add more information link

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

## ToDo

- [x] `applecamerad` (nothing better found then [this](https://www.theiphonewiki.com/wiki/Services) page instead of manual)

### CODE REVIEW

File: pages/osx/applecamerad.md
Comment (navarroaxel) [inline pages/osx/applecamerad.md:5]: This doesn't look useful.

Comment (EmilySeville7cfg) [reply pages/osx/applecamerad.md:5]: [applecamerad command man page](https://www.reddit.com/r/osx/comments/t7b1xg/applecamerad_command_man_page/) - my question

Comment (navarroaxel) [reply pages/osx/applecamerad.md:5]: Hard to find something specific.

**Review by navarroaxel — APPROVED** [review]

**Comment by CleanMachine1** [comment]
I'm inclined to agree with axel. This doesn't really add anything from what I can see. I don't know whether I approve, however if anyone else agrees with the link then just merge, its not a big deal!

**Comment by navarroaxel** [comment]
> I'm inclined to agree with axel. This doesn't really add anything from what I can see. I don't know whether I approve, however if anyone else agrees with the link then just merge, its not a big deal!

@CleanMachine1 We should add links to all the pages, this demons commands for macOS are a veteran level game.

**Review by blueskyson — COMMENTED** [review]
I personally don't approve this, either. This link contains few information of this command.

**Comment by navarroaxel** [comment]
@blueskyson but we want all the pages with links to turn on the lint rule as required. We don't have a better option.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**
